lib/src/Uncertainty/Bayesian/openturns/IndependentMetropolisHastings.hxx

```diff
@@ -0,0 +1,87 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief Metropolis-Hastings algorithm
+ *
+ *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_INDEPENDENTMETROPOLISHASTINGS_HXX
+#define OPENTURNS_INDEPENDENTMETROPOLISHASTINGS_HXX
+
+#include "openturns/OTprivate.hxx"
+#include "openturns/MetropolisHastingsImplementation.hxx"
+#include "openturns/ResourceMap.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class IndependentMetropolisHastings
+ *
+ * @brief Metropolis-Hastings algorithm.
+ *
+ */
+class OT_API IndependentMetropolisHastings
```

Comment:
you've got to mention new classes in Changelog file though

Reply:
Done